### PR TITLE
Merge images/README.md with the style guide

### PR DESCRIPTION
**Pull request by keirthana**
- Update guidance to store images in GH repo
- Merge images/README.md with the style guide

### CODE REVIEW

File: contribute/anbox-style-guide.md [hunk 1]
```diff
@@ -98,6 +98,187 @@ Use alternative text (or alt text) for images when visual details cannot be incl
...
+
+1. Go to **File** > **Open Library from** > **Device**.
+2. Select the [`anbox-cloud-docs/images/anbox-drawio.xml`](/images/anbox-drawio.xml) file.
+
+Configure custom styles and colours:
+
+1. Go to **Extras** > **Configuration**.
+2. Paste the following configuration:
+
+```
```
Comment (ajanon) [inline contribute/anbox-style-guide.md:119]:
````suggestion
```json
````
Just so we may get coloration.

**Review by ajanon — CHANGES_REQUESTED** [review]
One nitpick.

**Review by ajanon — APPROVED** [review]
LGTM, thanks!

**Review by morphis — APPROVED** [review]
LGTM!

**Pull request merged**